build.sbt

```diff
@@ -129,36 +122,16 @@ lazy val site = project
     ),
   }
 
-// General Settings
-lazy val commonSettings = Seq(
-  headerLicenseStyle := HeaderLicenseStyle.SpdxSyntax
-) ++ automateHeaderSettings(Compile, Test)
-
 val Scala213 = "2.13.7"
 val Scala213Cond = s"matrix.scala == '$Scala213'"
 
 // General Settings
 inThisBuild(
   List(
-    organization := "org.typelevel",
-    organizationName := "Typelevel",
-    publishGithubUser := "rossabaker",
-    publishFullName := "Ross A. Baker",
-    baseVersion := "1.1",
+    tlBaseVersion := "1.2",
     crossScalaVersions := Seq("2.12.15", Scala213, "3.0.2"),
-    scalaVersion := crossScalaVersions.value.filter(_.startsWith("2.")).last,
-    versionIntroduced := Map(
-      "3.0.0-RC1" -> "1.0.0",
-      "3.0.0-RC2" -> "1.0.1",
-      "3.0.0-RC3" -> "1.1.3"
-    ),
     homepage := Some(url("https://github.com/typelevel/case-insensitive")),
```

Comment:
This is set automatically too btw.

Reply:
True, but it should probably be https://typelevel.org/case-insensitive/.